app/src/main/java/com/github/sdp/tarjetakuna/utils/ResourceHelper.kt

```diff
@@ -0,0 +1,48 @@
+package com.github.sdp.tarjetakuna.utils
+
+import java.io.ByteArrayOutputStream
+import java.io.IOException
+import java.io.InputStream
+
+class ResourceHelper {

```

Comment:
This name is not very insightful. Did you plan to use it in other places in the future? Otherwise, maybe smth like "CardsFromApiLoader"?

Reply:
Yes other place in future. It s not at all a cardsFromApiLoader, but a helper to manage and read Assets locally from the assets folder. Nothing to do with the API